docs/_v15_x/2_send.md

```diff
@@ -255,6 +255,87 @@ client.createMessage(message)
         .send();
 ```
 
+### Datatype ShareDocumentsRequest
+
+This datatypes facilitates exchange of documents between an organisation and a Digipost end user. The sender
```

Comment:
```suggestion
This datatype can be used by organisations to share documents with an Digipost end user. The sender
```
"facilitate", too vague?
"exchange": too general? It is a one-way sharing process, not a mutual exchange- Exchange can even be misunderstood as _messaging_, in general.